core/inc/knoting/window.h

```diff
@@ -3,19 +3,34 @@
 #include <inttypes.h>
 #include <string>
 
+#include "knoting/subsystem.h"
```

Comment:
Let's decide in tomorrow's meeting if we want as convention `"knoting/subsystem.h"` or `<knoting/subsystem.h>`, since it is not in the same path, or a subdirectory of `src/`, this _include_ is looked up inside the _external include_ paths.